composer.json

```diff
@@ -58,7 +58,7 @@
             "drupal/cas": {
                 "Add support for http CAS servers": "https://www.drupal.org/files/issues/2018-09-03/2996924-4.patch",
                 "Provide custom mail handlers": "https://www.drupal.org/files/issues/2018-09-04/2996955-2.patch",
-                "Provide after validation event": "https://www.drupal.org/files/issues/2018-09-05/2997099-4.patch"
+                "Provide pre validation event": "https://www.drupal.org/files/issues/2018-09-28/3001882-11.patch"
```

Comment:
@imanoleguskiza The patch was merged, so we can remove it from here. 
Same goes for https://www.drupal.org/project/cas/issues/2996924

I think we can use the dev version of CAS, at least until we tag the first 1.0 release. 

@ademarco what do you think? Maybe we can ask him to make a release in the meantime? Last was Feb.